package.json

```diff
@@ -14,8 +14,8 @@
   ],
   "scripts": {
     "setup": "yarn install && yarn allow-scripts",
-    "test": "jest",
-    "test:watch": "jest --watch",
+    "test": "jest --maxWorkers=1",
+    "test:watch": "jest --maxWorkers=1 --watch",
```

Comment:
This is a workaround for Jest not supporting `bigint`s. For some reason it works when using `--maxWorkers=1`.

Reply:
Maybe newer jest version does work?

Reply:
Tried that, unfortunately not.